debian/mariadb-client-10.5.install

```diff
@@ -1,29 +1,17 @@
 debian/additions/innotop/innotop usr/bin/
 debian/additions/mysqlreport usr/bin/
 usr/bin/mariadb-conv
-usr/bin/mysql_find_rows
-usr/bin/mysql_fix_extensions
-usr/bin/mysql_waitpid
-usr/bin/mysqlaccess
-usr/bin/mysqladmin
-usr/bin/mysqldump
-usr/bin/mysqldumpslow
-usr/bin/mysqlimport
-usr/bin/mysqlshow
+usr/bin/mariadb-find-rows
+usr/bin/mariadb-fix-extensions
+usr/bin/mariadb-waitpid
+usr/bin/mariadb-access
+usr/bin/mariadb-admin
+usr/bin/mariadb-dump
+usr/bin/mariadb-dumpslow
+usr/bin/mariadb-import
+usr/bin/mariadb-slap
 usr/bin/mysqlslap
 usr/bin/mytop
-usr/share/man/man1/mariadb-access.1
```

Comment:
Why remove all the mariadb-* man pages?

Reply:
I didn't change the names of the man pages, therefore I create MariaDB named symlinks of them in .links and then there is no reason to install them a second time since .links are installed.

Reply:
ok, but maybe also rename the man pages in a later commit some day?